pkg/cloud/ibmcloud/clients/machines.go

```diff
@@ -226,11 +225,14 @@ func waitTransactionDone(s *services.Virtual_Guest) error {
 		s.Session.Debug = true
 	}
 
+	klog.Infof("Waiting to get active transactions in %ds", options.WaitTransactionTimeout)
```

Comment:
Please lower the debug level, it is not necessary to display it in every time for guest waiting.

Reply:
According to other codes, is klog.V(4) ok for this debug message?